### PR TITLE
Fix French i18n

### PR DESCRIPTION
**Pull request by yannickguillemot**
Some of you may know, I am raised and born in France. I wanted to help fixing a bit some weird translations I noticed in the i18n manifest for the french object. Here's an exhaustive list of my work. Pardon the lack of french accentuation, I feel lazy right now but be sure that accuentation is correct in the code. :)

- 'Cooldown' translates better into 'Regeneration'. I checked the Armory in French and it is the term used.
- 'Tag' is not a verb in french, 'Tagger' is commonly used.
- For 'Junk': 'Jonque' is an old word not known by our generation, 'Camelote' is more modernly used.
- 'Equipé' will be conjugated to 'Loadout' -> masculine.
- 'Article Nivellement' does not make sense.
- 'Recueillir des Engrammes': Knowing DIM and understanding the use case, I'd say that my translation would be clearer.
- Farming mode: 'Agriculture' is really used for farmers in real life, not applicable in gaming. French gamers say 'farm' often. Therefore, 'mode Farm' made sense. Leaving the english term capitalized will help them understand it is a use of an english word.
- Added translations for loadout application.

Feel free to comment. And I'd love to comment with another frenchie if there is one around here. :)

### CODE REVIEW

File: app/scripts/dimApp.i18n.js [hunk 1]
```diff
@@ -152,33 +152,37 @@
...
+          'Item Leveling': "Evolution d'article",
+          'Gather Engrams': "Rassembler les Engrammes",
           gather_engrams_except_exotics: "Exotiques",
-          farming_mode: "Engrammes Coffres (Agriculture)",
+          farming_mode: "Engrammes aux Coffres (mode Farm)",
           restore_all_items: "Tous les Articles",
-          Loadouts: "Loadouts"
+          Loadouts: "Loadouts",
+          LoadoutAppliedAuto: "Constructeur de Loadout automatique",
+          LoadoutApplied: "Votre loadout de {{amount}} articles a été transféré à votre {{store}}",
```
Comment (bhollis) [inline app/scripts/dimApp.i18n.js]: It'll be a character name ("female exo warlock" in English)

**Comment by bhollis** [comment]
This is awesome! Thanks so much for updating these. Would you mind merging from master?

File: app/scripts/dimApp.i18n.js (hunk 1 above)
Comment (yannickguillemot) [inline app/scripts/dimApp.i18n.js]: Ok so I need to change that sentence a bit. I will do it tonight.

**Comment by yannickguillemot** [comment]
@bhollis Merge FROM my fork's master or INTO DIM's master?

**Comment by bhollis** [comment]
Merge dim into your fork so the conflict gets solved.

**Comment by yannickguillemot** [comment]
Rebased off of DIM:dev.

**Comment by bhollis** [comment]
Wonderful, thanks!

**Pull request merged**